finalPrep/annotations.js

```diff
@@ -14,29 +18,36 @@ function buildABear(name, age, fur, clothes, specialPower) {
     sayings: [greeting, powerSaying, "Goodnight my friend!"],
     isCuddly: true,
   };
-
+// return the result of the object
```

Comment:
I would not go with the word `result` here... this isn't really an operation, like math, but rather just returning the variable that was just declared. You can go through the declared object line by line as well- it is helpful to note if the values come from a previously declared variable, or a parameter.